index.js

```diff
@@ -249,6 +260,14 @@
     toString: always('???')
   };
 
+  //  Inconsistent :: Type
+  var Inconsistent = {
+    '@@type': 'sanctuary-def/Inconsistent',
+    type: 'INCONSISTENT',
+    test: K(false),
+    toString: always('???')
+  };
+
```

Comment:
Whereas `[] :: Array Unknown`, `['foo', false] :: Array Inconsistent`. Using the same internal representation for both types was problematic since we need to be able to differentiate the two cases.